pkg/kube/kube.go

```diff
@@ -58,3 +58,12 @@ func ConfigMapValue(client client.Client, namespacedName types.NamespacedName, k
 	}
 	return string(data), nil
 }
+
+func PostgreSQLDatabases(c client.Client, namespace string) ([]lunarwayv1alpha1.PostgreSQLDatabase, error) {
+	var databases lunarwayv1alpha1.PostgreSQLDatabaseList
+	err := c.List(context.TODO(), &databases, client.InNamespace(namespace))
```

Comment:
TODO is that a real TODO or just a weird name?

Reply:
This is in fact a function of the `context` package 🙈 
https://godoc.org/context#TODO

Reply:
Of God 😂 
`Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter).`

I guess it is kind of TODO-ish.

Reply:
Indeed 😄